pkg/apis/ceph.rook.io/v1/pool.go

```diff
@@ -53,34 +53,47 @@ func (p *ReplicatedSpec) IsTargetRatioEnabled() bool {
 func (p *CephBlockPool) ValidateCreate() error {
 	logger.Infof("validate create cephblockpool %v", p)
 
-	err := validatePoolSpec(p.Spec.ToNamedPoolSpec())
+	err := ValidateCephBlockPool(&p.Spec)
 	if err != nil {
 		return err
 	}
 	return nil
 }
 
+// ValidateCephBlockPool validates specifically a CephBlockPool's spec (not just any NamedPoolSpec)
+func ValidateCephBlockPool(s *NamedBlockPoolSpec) error {
+	name := s.Name
+	if name == "device_health_metrics" || name == ".mgr" || name == ".nfs" {
```

Comment:
nit: why use a local var?
```suggestion
	if s.Name == "device_health_metrics" || s.Name == ".mgr" || s.Name == ".nfs" {
```